plugins/azure/resoto_plugin_azure/resource/base.py

```diff
@@ -18,6 +18,13 @@
 from resotolib.types import Json
 
 log = logging.getLogger("resoto.plugins.azure")
+
+
+def get_client(subscription_id: str) -> AzureClient:
+    credential = AzureAccountConfig().credentials()
+    return AzureClient.create(credential=credential, subscription_id=subscription_id)
```

Comment:
Requires import: 
from resotolib.config import current_config
```suggestion
    config = current_config()
    azure_config = cast(AzureAccountConfig, config.azure)
    return AzureClient.create(credential=azure_config.credentials(), subscription_id=subscription_id)
```

Instantiating a config object will not use the configured values!